src/main.tsx

```diff
@@ -11,6 +11,7 @@ import { isMac } from "./utils";
 function main() {
   const pluginId = logseq.baseInfo.id;
   console.info(`#${pluginId}: MAIN`);
+  parent.document.body.classList.add('is-tabs');
```

Comment:
This will decorate the root view's body class with `.is-tabs` class name, right? I think you may want to make it longer to make it more easily to be identified for ease of maintenance, e.g., `is-plugin-tabs-enabled`, or a data attribute like `data-plugin-tabs="enabled"`.

Reply:
> This will decorate the root view's body class with .is-tabs class name, right? 

yes!

>  I think you may want to make it longer

done